tests/Test/TestLoggerTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Psr\Log\Tests\Test;
+
+use Psr\Log\Test\LoggerInterfaceTest;
+use Psr\Log\Test\TestLogger;
+
+/**
+ * Test classes from psr/log 1.1.x
+ *
+ * @requires PHP 7.4
+ * @covers \Psr\Log\Test\TestLogger
+ * @covers \Psr\Log\Test\LoggerInterfaceTest
+ *
+ * @author Jérôme Tamarelle <jerome@tamarelle.net>
+ */
+class TestLoggerTest extends LoggerInterfaceTest
```

Comment:
Shouldn't this just extend the class from the `psr/log` package? So once ported, it will automatically switch to use the class from `src` instead that from vendor.

Reply:
That's precisely what is done. The class `Psr\Log\Test\LoggerInterfaceTest` does not exist in this package v1.0. It is loaded from `psr/log`.

I've added assertions for specific features of `Psr\Log\Test\TestLogger`.